library/include/ck/library/tensor_operation_instance/gpu/gemm_add_relu_add_layernorm.hpp

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <cstdlib>
```

Comment:
Remove this including since `cstdlib` is not used here,  also remove the same including in other similar `.hpp` files

Reply:
Done in b710aed463901bad05d000e4dbf0d8968443a124